_includes/newcomers/newcomers.html

```diff
@@ -0,0 +1,89 @@
+<h3>Welcome Newcomers!</h3>
+
+<p>
+    Whether you are a first-time contributor or an open-source veteran, you are welcome to contribute to and actively
+    engage in the development of projects at Layer5.
+</p>
+<p>
+    Here are some instructions to get you started and if you haven't yet, join the Slack channel to collaborate with the
```

Comment:
Add the Slack channel link here please. Btw, awesome work @navendu-pottekkat . You have done some heavy lifting here.

Reply:
Sure. I have missed a lot of links. I will fix it. And thank you @tripathyprateek